### ISSUE TITLE
Conflicts between twitter_markov and pyyaml

### ISSUE THREAD

**Issue by NeolithEra** [fitnr/twitter_markov#14]
Hi, users are unable to run **_twitter_markov_** due to dependency conflict with _**pyyaml**_ package. As shown in the following full dependency graph of **_twitter_markov_**, **_twitter_markov_** requires **_pyyaml *_**，while **_twitter-bot-utils_** requires **_pyyaml ==3.11_**.

According to pip’s _“first found wins”_ installation strategy, _**pyyaml 5.2**_ is the actually installed version. However, _**pyyaml 5.2**_ does not satisfy **_==3.11_**.

### Dependency tree-----------
```
twitter-markov - 0.5.1
| +- markovify(install version:0.3.1 version range:<0.4,>=0.2.4)
| | +- unidecode(install version:1.1.1 version range:*)
| +- python-levenshtein(install version:0.12.0 version range:<0.13,>=0.12.0)
| | +- setuptools(install version:42.0.2 version range:*)
| +- pyyaml(install version:5.2 version range:*)
| +- six(install version:1.13.0 version range:*)
| +- tweepy(install version:3.8.0 version range:*)
| | +- pysocks(install version:1.7.1 version range:>=1.5.7)
| | +- requests(install version:2.22.0 version range:>=2.11.1)
| | | +- certifi(install version:2019.11.28 version range:>=2017.4.17)
| | | +- chardet(install version:3.0.4 version range:<3.1.0,>=3.0.2)
| | | +- idna(install version:2.8 version range:>=2.5,<2.9)
| | | +- urllib3(install version:1.25.7 version range:<1.26,>=1.21.1)
| | +- requests-oauthlib(install version:1.3.0 version range:>=0.7.0)
| | +- six(install version:1.13.0 version range:>=1.10.0)
| +- twitter-bot-utils(install version:0.11.6.post1 version range:<0.12,>=0.11.6.post1)
| | +- pyyaml(install version:3.11 version range:==3.11)
| | +- tweepy(install version:3.8.0 version range:<4,>=3.5.0)
| | | +- pysocks(install version:1.7.1 version range:>=1.5.7)
| | | +- requests(install version:2.22.0 version range:>=2.11.1)
| | | | +- certifi(install version:2019.11.28 version range:>=2017.4.17)
| | | | +- chardet(install version:3.0.4 version range:<3.1.0,>=3.0.2)
| | | | +- idna(install version:2.8 version range:>=2.5,<2.9)
| | | | +- urllib3(install version:1.25.7 version range:<1.26,>=1.21.1)
| | | +- requests-oauthlib(install version:1.3.0 version range:>=0.7.0)
| | | +- six(install version:1.13.0 version range:>=1.10.0)
| +- wordfilter(install version:0.2.6.2 version range:>=0.1.8)
```

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Suggested Solution

1.	Fix your direct dependency to be **_pyyaml ==3.11_**. I have checked this revision will not affect your downstream projects now.
2.	Ask your upstream project **_twitter-bot-utils_** to loose the version range of **_pyyaml_**.
3.	Remove your direct dependency **_pyyaml_**, and use **_pyyaml_** transitively introduced by **_twitter-bot-utils_**.

@fitnr Which solution do you prefer, 1, 2 or 3?
Please let me know your choice. I can submit a PR to solve this issue.

**Comment by fitnr** [comment]
Is this a bot? Amazing. I pick choice 2.

**Comment by fitnr** [comment]
Closed by v0.6.0 release
